src/libs/ReportUtils.ts

```diff
@@ -4102,6 +4102,45 @@ function buildOptimisticReportPreview(chatReport: OnyxEntry<Report>, iouReport:
     };
 }
 
+/**
+ * Builds an optimistic ACTIONABLETRACKEXPENSEWHISPER action with a randomly generated reportActionID.
+ */
+function buildOptimisticActionableTrackExpenseWhisper(iouAction: OptimisticIOUReportAction, transactionID: string): ReportAction {
+    const currentTime = DateUtils.getDBTime();
+    const targetEmail = CONST.EMAIL.CONCIERGE;
+    const actorAccountID = PersonalDetailsUtils.getAccountIDsByLogins([targetEmail])[0];
+    const reportActionID = NumberUtils.rand64();
+    return {
+        actionName: CONST.REPORT.ACTIONS.TYPE.ACTIONABLE_TRACK_EXPENSE_WHISPER,
+        actorAccountID,
+        avatar: UserUtils.getDefaultAvatarURL(actorAccountID),
+        created: DateUtils.addMillisecondsFromDateTime(currentTime, 1),
+        lastModified: DateUtils.addMillisecondsFromDateTime(currentTime, 1),
+        message: [
+            {
+                html: 'What would you like to do with this expense?',
+                text: 'What would you like to do with this expense?',
```

Comment:
Don't we need to localise the text `What would you like to do with this expense?` [here](https://github.com/Expensify/App/blob/97097ff13b086c3d44d26d44335921323b211669/src/languages/en.ts#L2839) which seems correct for our context? Also, I found another reference of the same text [here](https://github.com/Expensify/App/blob/957b5ec8cd0803b1e7f878aabf6ddff7eeaabab5/src/pages/home/report/ContextMenu/ContextMenuActions.tsx#L362) which can be replaced with localized string.

Reply:
There are four references to the same text in code which is not good. So, if we are not going ahead without translation, let us, at least, set the text as a CONST value. 
But, let us first get a reconfirmation of whether we need translation here or not. 

@marcochavezf  
This is regarding the text `What would you like to do with this expense?` which is also sent by BE and it seems to not have a translation in BE.
So I am curious to know if we need a translation for this. 
What’s your take on this?

Reply:
There are four references to the same text in code which is not good. So, if we are not going ahead without translation, let us, at least, set the text as a CONST value. 
But, let us first get a reconfirmation of whether we need translation here or not. 

@marcochavezf  
This is regarding the text `What would you like to do with this expense?` which is also sent by BE and it seems to not have a translation in BE.
So I am curious to know if we need a translation for this. 
What’s your take on this?

Reply:
Hi @tienifr @rojiphil, sorry for the delay here I was OOO last week.

This one is tricky because as @tienifr mentioned, we don't translate from BE for now. So it would be inconsistent if we translate optimistically in the frontend and then the backend responds with the message in English. So we can leave it as is for now

Reply:
Thanks @marcochavezf. 
@tienifr Since no translation is needed here, let us move this text to a CONST value to avoid repetitive text. The same text is also used [here](https://github.com/Expensify/App/blob/2514f2988eb26858d30fa501c0040b1f8fc2e239/src/pages/home/report/ContextMenu/ContextMenuActions.tsx#L364) which can be replaced with a CONST value.